### PR TITLE
updated templates and scripts to create default as well as visibility…

### PR DESCRIPTION
**Pull request by ktenzer**
… databases

Signed-off-by: ktenzer <keith.tenzer@gmail.com>

### CODE REVIEW

**Review by alexandrevilain — CHANGES_REQUESTED** [review]
Hi! Thanks for your PR! 👍 
LGTM for sql databases, but I think this breaks cassandra persistence reconciliation.

If we look at `temporal-cassandra-tool` it does not support `create-database`.
We have to handle the cassandra case, with something like:

```
{{ .Tool }} {{ .ConnectionArgs }}  create-keyspace -keyspace {{ .Keyspace }}
```

See: https://github.com/temporalio/temporal/blob/master/tools/cassandra/main.go#L204

**Comment by ktenzer** [comment]
I am now handling mysql/postgres vs cassandra db create. Great catch! I also tested both cassandra and postgres and working as expected.

Cassandra
```[ktenzer@fedora temporal-operator]$ kubectl get pods -n temporal-test
NAME                                           READY   STATUS      RESTARTS      AGE
cassandra-0                                    1/1     Running     0             26m
prod-create-default-database-gbs8c             0/1     Completed   0             7m45s
prod-create-visibility-database-h8gjr          0/1     Completed   0             7m43s
prod-frontend-677465844f-lbpms                 1/1     Running     0             7m21s
prod-history-d5d95477-z2ndb                    1/1     Running     0             7m20s
prod-matching-7fbbf87bd4-6dqdc                 1/1     Running     0             7m20s
prod-setup-default-schema-tffhs                0/1     Completed   0             7m41s
prod-setup-visibility-schema-r74dl             0/1     Completed   0             7m38s
prod-update-default-schema-v-1-17-5-csvp7      0/1     Completed   0             7m35s
prod-update-visibility-schema-v-1-17-5-z4btn   0/1     Completed   0             7m26s
prod-worker-6c87b56595-97bd9                   1/1     Running     2 (80s ago)   7m20s```

Postgres
```
[ktenzer@fedora temporal-operator]$ kubectl get pods -n temporal-test
NAME                                           READY   STATUS      RESTARTS   AGE
prod-create-default-database-whd9g             0/1     Completed   0          84s
prod-create-visibility-database-hfpmb          0/1     Completed   0          82s
prod-frontend-5dd78d5699-wtt4k                 1/1     Running     0          70s
prod-history-79d5d4d699-dsjrf                  1/1     Running     0          70s
prod-matching-59c58ff8f4-884v5                 1/1     Running     0          69s
prod-setup-default-schema-n562z                0/1     Completed   0          79s
prod-setup-visibility-schema-8s22x             0/1     Completed   0          77s
prod-update-default-schema-v-1-17-5-qfswg      0/1     Completed   0          75s
prod-update-visibility-schema-v-1-17-5-stxps   0/1     Completed   0          73s
prod-worker-7cb5f9df5c-cjt2r                   1/1     Running     0          69s
```

File: pkg/resource/persistence/schema_scripts_configmap_builder.go
Comment (alexandrevilain) [inline pkg/resource/persistence/schema_scripts_configmap_builder.go:288]:
```suggestion
			Tool:           visibilityStoreTool,
```

File: pkg/resource/persistence/schema_scripts_configmap_builder.go
Comment (alexandrevilain) [inline pkg/resource/persistence/schema_scripts_configmap_builder.go:298]:
```suggestion
			Tool:           visibilityStoreTool,
```

**Review by alexandrevilain — CHANGES_REQUESTED** [review]
Only two minor changes due to a copy/past error. It handles the case if the visibility store uses another database than the default store. 

Once fixed, I'll merge it! Thanks :)

File: pkg/resource/persistence/schema_scripts_configmap_builder.go
Comment (alexandrevilain) [inline pkg/resource/persistence/schema_scripts_configmap_builder.go:290]:
```suggestion
			KeyspaceName:   visibilityStore.Cassandra.Keyspace,
```

File: pkg/resource/persistence/schema_scripts_configmap_builder.go
Comment (ktenzer) [reply pkg/resource/persistence/schema_scripts_configmap_builder.go:298]: now i look like a chump so much for first impressions

Comment (alexandrevilain) [reply pkg/resource/persistence/schema_scripts_configmap_builder.go]: No problem, you're new to the codebase, which is perhaps a bit complicated. 
As I'm currenlty the only maintainer, the current codebase lacks of code reviews which inevitably makes the code harder to understand for others :)

**Review by alexandrevilain — APPROVED** [review]

**Comment by alexandrevilain** [comment]
Hey @ktenzer you're now an offical contributor of this operator 🎉
Thanks you for your work 😀

**Comment by alexandrevilain** [comment]
FYI I will release v0.6.1 with your changes tomorrow :)

**Pull request merged**
